### ISSUE TITLE
FIrebase iOS integration crash – caused by an invalid GOOGLE_APP_ID

### ISSUE THREAD

**Issue by xieruimoonbee** [LLK/scratchjr#231]
Xcode version is 11.2.1
i can build  successful. but luanching one second has crash
i tried move GoogleService-Info.plist to program root folder but  it's still the same mistake.
opens GoogleService-Info.plist found GOOGLE_APP_ID is empty , i don't know if it's right

**Comment by apple502j** [comment]
Maybe you need your own app?

**Comment by xieruimoonbee** [comment]
@apple502j You are right! we can delete this firebase with APP delegate ,thanks!!!!!! om my god ,i'm stupid
